docs/help-community/contributing.md

```diff
@@ -52,53 +52,20 @@ You can contribute to the n8n documentation, for example by documenting nodes or
 
 The repository for the docs is [here](https://github.com/n8n-io/n8n-docs){:target=_blank class=.external-link} and the guidelines for contributing to the docs are [here](https://github.com/n8n-io/n8n-docs/blob/master/CONTRIBUTING.md){:target=_blank class=.external-link}.
 
-## Write a blog post
+## Contribute to the blog
 
-You can write an article for the [n8n blog](https://n8n.io/blog/){:target=_blank class=.external-link}. Your article can be, for example, a [workflow tutorial](https://n8n.io/blog/tag/tutorial/){:target=_blank class=.external-link}, an opinion piece on automation, or some domain-specific [automation ideas](https://n8n.io/blog/tag/ideas/){:target=_blank class=.external-link}. 
+You can write an article for the [n8n blog](https://blog.n8n.io/){:target=_blank class=.external-link}. Your article can be, for example, a [workflow tutorial](https://blog.n8n.io/tag/tutorial/){:target=_blank class=.external-link}, an opinion piece on automation, or some domain-specific [automation guides](https://blog.n8n.io/tag/guide/){:target=_blank class=.external-link}.
 
-### Process to submit a post
+### How to submit a post
 
-n8n appreciates all contributions. Publishing a tutorial on your own site that supports the community is a great contribution. However, if you want n8n to highlight your post on our blog, you need to go through this process.
+n8n appreciates all contributions. Publishing a tutorial on your own site that supports the community is a great contribution. If you want n8n to highlight your post on the blog, you need to go through the following steps: 
```

Comment:
```suggestion
n8n appreciates all contributions. Publishing a tutorial on your own site that supports the community is a great contribution. If you want n8n to highlight your post on the blog, follow these steps: 
```

Reply:
this looks great, I've committed your change — could you review & merge? @freakwriter